### ISSUE TITLE
Grouping by Date (day, week, month, year ...)

### ISSUE THREAD

**Issue by Yousskill** [odoo/odoo#39418]
Impacted versions: 12

Steps to reproduce: 

when i write this code:

`<filter name="groupby_delivery_month" context="{'group_by': 'delivery_date:month'}" string="Mois de livraison" help="Mois de livraison"/>`

this should give me a different result than: 

`<filter name="groupby_delivery_date" context="{'group_by': 'delivery_date'}" string="Date de livraison" help="Date de livraison"/>`

Current behavior:

I always see the black arrow beside my filter whatever type of date filtering i write in my code (day, week, month, year ...)

![image](https://user-images.githubusercontent.com/33576296/67665749-a7e28280-f96a-11e9-8ada-c0c09cbada4b.png)

Expected behavior:

I must see the same picture but without the black arrow and when i click on it, this should group my records directly by month.

**Comment by pedrobaeza** [comment]
That's the expected behavior: you define what is the default grouping interval in the XML, that's to say, when clicking directly on the name of the filter, but from v12, you *always* have the possibility of grouping by any of the date intervals through the black arrow.

Closing as no issue, but expected behavior.

**Comment by Yousskill** [comment]
But I don't have the expected behavior even if i click directly on the name of the filter, it always force me to choose a type like if i didn't add the :week keyword, are you sure this is how it has to be?

**Comment by pedrobaeza** [comment]
Yeah, that can be something unexpected... reopening and tagging for gathering opinions on this.

**Comment by LuisOseguera** [comment]
There isn't a solution yet? I'm having the same issue and I'd like to know an alternative or another solution.

![odoo-group-issue](https://user-images.githubusercontent.com/47401069/167183502-8313f2e6-b951-4ba9-ab0a-5423559ce6a4.gif)
